Makefile

```diff
@@ -20,7 +20,7 @@ SHELL:=/usr/bin/env bash
 
 .DEFAULT_GOAL:=help
 
-GO_VERSION ?= 1.20.4
+GO_VERSION ?= 1.20.8
```

Comment:
This variable is only used to echo a version on the last line of makefile, so it is not bumping Go.

Go is bumped in Dockerfile.

/hold